modules/common-workflows/pages/workflow-removing-channel.adoc

```diff
@@ -0,0 +1,150 @@
+[[workflow-removing-channel]]
+= Removing channel
+
+
+== Use case
+
+This workflow shows how to manually remove SUSE provided channels from {productname} and to clean up end-of-life products, to reclaim space or for other purposes.
+
+
+
+== Step-by-step instructions
+
+Manual channel removal consists of the following phases:
+
+. <<preparing-to-remove-channel>>
+. <<removing-channels>>
+. <<cleaning-up-channel-packages>>
+
+
+[[preparing-to-remove-channel]]
+=== Preparing to remove a channel
+
+Before removing a channel, you must identify the label for the channel to be removed.
+It can be done with {webui} or on the command line.
+
+[IMPORTANT]
+====
+* Be careful not to remove channels that systems are currently subscribed to, or that you are planning on subscribing a system to.
+* If there are systems that are currently subscribed to a channel that you want to delete, do not remove the channel until those systems are upgraded or unsubscribed.
+====
+
+
+==== Identifying Channel labels
+
+.Procedure: Identifying channel labels using {webui}
+[role=procedure]
+. In the SUSE Manager {webui}, in the sidebar navigate to menu:Software[Channel List > All]
```

Comment:
```suggestion
. In the {productname} {webui}, navigate to menu:Software[Channel List > All].
```
The next "steps" are not always steps, and the procedure sounds unnecessarily complex.  I'd vote to do the thing first completely with the webui and then the complete thing on the command line.